### DISCUSSION TITLE
Neighbor Solicitation

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#30] [General]
Hi Brian,
In the second paragraph of "Address resolution", you claim that NS is sent to an all-nodes multicast address.
I do not agree. NS is sent to solicited-node multicast. Not every node is interrupted by this request (albeit collision is possible).
Ed/

**Comment by becarpenter** [comment]
Correct, thanks, will fix.

(BTW you could raise such errors as Issues not Discussions in GitHub.)

**Comment by becarpenter** [comment]
Closed as done
